morphia/src/main/java/org/mongodb/morphia/DatastoreImpl.java

```diff
@@ -250,6 +260,40 @@ public void ensureCaps() {
     }
 
     @Override
+    public void enableDocumentValidation() {
+        for (final MappedClass mc : mapper.getMappedClasses()) {
+            Validation validation = (Validation) mc.getAnnotation(Validation.class);
+            if (validation != null) {
+                String collectionName = mc.getCollectionName();
+                CommandResult result = getDB()
+                    .command(new BasicDBObject("collMod", collectionName)
+                                 .append("validator", Document.parse(validation.value()))
```

Comment:
Mostly because I didn't realize it was there.  I'd used JSON.parse() in the past and it failed on certain docs.  I knew of Document.parse() and went for that.  remove all the Document references.